### PR TITLE
PR for Like and bookmark

### PR DESCRIPTION
**Pull request by SK02K1**
### Features

- Like post
- Unlike liked post.
- Add post to bookmarks.
- Remove post from bookmarks.

[Deploy Preview](https://deploy-preview-4--sharemoment.netlify.app/login)

### CODE REVIEW

File: src/app/features/posts/postsSlice.js [hunk 1]
```diff
@@ -68,6 +68,30 @@ export const editPost = createAsyncThunk(
...
+        }
+      );
+      if (status === 201) {
+        return {
+          likesData: data.posts.find(({ _id }) => _id === postID).likes,
+          postID,
+        };
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (manojasarsa) [inline src/app/features/posts/postsSlice.js]: you can use `console.error` here

**Review by manojasarsa — APPROVED** [review]
The like/dislike and add/remove bookmark is working as expected.
The code seems clean and structured. 
The UI is looking amazing.
Good job!

**Review by manismk — APPROVED** [review]
Good Naming convention and modular code.
Functionality works as expected on the Home page.
But in single post page like/unlike feature is not working.

File: src/app/features/posts/postsSlice.js (hunk 1 above)
Comment (SK02K1) [reply src/app/features/posts/postsSlice.js]: thanks for the suggestion

**Pull request merged**